volumes/idempotency.go

```
@@ -150,3 +150,14 @@ func (s *IdempotentService) Detach(ctx context.Context, volume *csi.Volume, serv
 		return err
 	}
 }
+
+func (s *IdempotentService) Resize(ctx context.Context, volume *csi.Volume, size int) error {
+	switch err := s.volumeService.Resize(ctx, volume, size); err {
+	case ErrNotAttached:
+		return nil
```

Why should errors be ignored when trying to resize a not attached volume?